src/main/java/net/minecraftforge/common/extensions/IForgeItemStack.java

```diff
@@ -178,6 +181,25 @@ default int getItemEnchantability()
         return self().getItem().getItemEnchantability(self());
     }
 
+    /**
+     * Gets the level of the enchantment currently present on the stack. Equivalent to calling {@link EnchantmentHelper#getItemEnchantmentLevel(Enchantment, ItemStack)}
+     * @param enchantment  the enchantment being checked for
+     * @return  Level of the enchantment, or 0 if not present
+     */
+    default int getEnchantmentLevel(Enchantment enchantment)
+    {
+        return self().getItem().getEnchantmentLevel(self(), enchantment);
+    }
+
+    /**
+     * Gets a map of all enchantments present on the stack.
+     * Use in place of {@link EnchantmentHelper#getEnchantments(ItemStack)} for gameplay enchantments, as this enforces the enchantment is not from an enchanted book.
+     */
```

Comment:
This could use having the javadocs updated to match the newer javadocs on the methods added in IForgeItem